tcl/envmngt.tcl.in

```diff
@@ -1166,27 +1166,42 @@ proc get-env {var {valifunset {}}} {
    }
 }
 
+proc isProtectedEnvVar {var} {
+   # return if var is protected by protected_envvars config
+   foreach protectedvar [split [getConf protected_envvars] :] {
+      if {$var eq $protectedvar} {
+         reportDebug "$var not set (protected_envvars config)"
+         return true
+      }
+   }
+   return false
```

Comment:
should simply return the result obtained from test:
`return [expr {$var in [split [getConf protected_envvars] :]}]`

Reply:
Okay, but what about `reportDebug` call?
Should I shift it to `set-env` and `unset-env` statements?

Reply:
no need, you already have the reportWarning there